.github/CODEOWNERS

```diff
@@ -300,6 +300,7 @@
 
 
 # Navigation bar project.
+/core/templates/domain/bottom_navbar/ @nithusha21 @showtim3
```

Comment:
Why bottom_navbar?

Reply:
It's actually a bottom navbar or should i call it change it to something else?

Reply:
Sorry earlier I was thinking this to be a domain object and my comment is incomplete, please ignore it! (sorry!)